### PR TITLE
[888] active section for navigation bar

### PR DESCRIPTION
**Pull request by gpeng**
### Context

We have a nav bar. It has links. They need a blue bar under them if they're active.

### Changes proposed in this pull request

Add active link styling when the page is current or the link is explicitly marked as current. This allows us to underline 'Trainees' for all trainee pages.

### CODE REVIEW

**Comment by avinhurry** [comment]
[Active section for navigation bar](https://trello.com/c/BTdxPnJo/888-active-section-for-navigation-bar)

File: app/helpers/navigation_helper.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module NavigationHelper
+  def trainee_link_is_current?
+    url = request.path_info
+    if url.include?("/trainees")
+      true
+    else
+      false
+    end
```
Comment (AbigailMcP) [inline app/helpers/navigation_helper.rb:10]:
```suggestion
    url.include?("/trainees")
```

**Review by AbigailMcP — COMMENTED** [review]
Looks good! Just one tidy-up point.

**Comment by AbigailMcP** [comment]
May be worth noting that at the moment I can be authenticated _and_ access the root page, so the navigation appears and looks funky. This doesn't need to be addressed in this ticket, but do we need to redirect authenticated users somewhere? @gpeng 

Screenshot 2021-01-20 at 11 01 18

File: app/helpers/navigation_helper.rb (hunk 1 above)
Comment (avinhurry) [reply app/helpers/navigation_helper.rb:10]: Thanks! That's cool.

**Review by AbigailMcP — APPROVED** [review]
🚢

**Comment by avinhurry** [comment]
> May be worth noting that at the moment I can be authenticated _and_ access the root page, so the navigation appears and looks funky. This doesn't need to be addressed in this ticket, but do we need to redirect authenticated users somewhere? @gpeng
> 
> Screenshot 2021-01-20 at 11 01 18

Discussed offline. Will serve `/home` from the root if the user is signed in. (As part of a separate PR.)

**Pull request merged**